### DISCUSSION TITLE
FR: When default author and post author are identical except for affiliation, allow override rather than duplicate

### DISCUSSION THREAD

**Discussion by swiderskiji** [quarto-dev/quarto-cli#10869] [Feature Requests]
### Description

I am creating a blog/site where I am the only author, so for the most part it makes sense for me to set a default author in `metadata.yml`, and I have.  But I'm not an academic and for the most part, I am writing only as an individual, not on behalf of or related to my employer or any other affiliation, so my default author doesn't have any affiliation attached:
```author:
  - name:
      given: J. I.
      family: Swiderski
    url: https://jski.net/
```

However, for some posts, for example where I am sharing documents written with and as public statements of an organization I'm part of, I would like to have such affiliation listed with the post:
```
author:
  - name:
      given: J. I.
      family: Swiderski
affiliation: Example Advisory Council
affiliation-url: http://example.com
```

However, when I try this, I end up with one author -- with no affiliation -- at the top, and two authors in the citation data at the bottom:
`  author = {Swiderski, J. I. and Swiderski, J. I.},`

(I wonder if this might be a side-effect of the solution to #8860...?)

Is it possible, or could it be possible, to have an affiliation listed on some posts and not others without having to enter the author name information on every post?

thanks,

**Comment by mcanouil** [comment]
Could you share a **small self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? The goal is to make it as easy as possible for us to recreate your problem so that we can fix it: please help us help you! Thanks.

Note that both authors and affiliations have "id", to allow reuse: https://quarto.org/docs/journals/authors.html
I suggest to go through the authors/affiliations and come back with a concrete example, likely as a Git repository in this case.

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

{{< lipsum 1 >}}

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image]({{< placeholder 600 400 >}}){#fig-placeholder}

{{< lipsum 1 >}}

The end after @fig-placeholder.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.
